packages/signature-controller/package.json

```diff
@@ -32,7 +32,7 @@
     "@metamask/approval-controller": "^4.0.0",
     "@metamask/base-controller": "^3.2.3",
     "@metamask/controller-utils": "^5.0.2",
-    "@metamask/logging-controller": "^1.0.3",
+    "@metamask/logging-controller": "^1.0.4",
```

Comment:
It looks like we are missing changelog entries for these changes. (Makes sense — these are easy to miss. We will have these automatically generated these soon.)

I will push up a new commit for this.